### PR TITLE
 Hopefully fixes tiles trying to be removed before the scene is loaded 

### PR DESCRIPTION
**Pull request by Bod9001**
yeah, this could have problems with tile being destroyed then place down, then it finally been done on client destroying the thing that had been placed down ,  should probably work this into update tile

### CODE REVIEW

**Comment by corp-0** [comment]
aaaaah you could have mentioned you were working on this, let me abort the oingoing build

**Comment by Bod9001** [comment]
it's not a build if it doesn't get cancelled and then run again a minute later

File: UnityProject/Assets/Scripts/Messages/Server/RemoveTileMessage.cs [hunk 1]
```diff
@@ -8,11 +9,57 @@ public class RemoveTileMessage : ServerMessage
...
+			RemoveAll = inRemoveAll;
+			TileChangeManager = inTileChangeManager;
+		}
+	}
 	public override void Process()
 	{
 		LoadNetworkObject(TileChangeManager);
-		var tileChangerManager = NetworkObject.GetComponent<TileChangeManager>();
-		tileChangerManager.InternalRemoveTile(Position, LayerType, RemoveAll);
+		if (NetworkObject == null)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Messages/Server/RemoveTileMessage.cs:31]: duplicated check

**Comment by corp-0** [comment]
what's up with the old commits?

**Comment by Bod9001** [comment]
odd, usually they don't show up it's because it's done on the same branch, but the commits should already be there so don't
 get it

**Comment by corp-0** [comment]
yeah, it is because you were using the same branch for all three of those changes. Anyways, it is fine. I will squash.

**Pull request merged**